.eslintrc.js

```
@@ -0,0 +1,37 @@
+module.exports = {
```

ESLint

[error] 'module' is not defined.

(no-undef)